src/ocamlorg_frontend/pages/releases.eml

```diff
@@ -13,69 +13,74 @@ Layout.render
-                <div class="flex flex-col items-center justify-center pt-24 space-y-4 text-lighter">
-                    <div>
-                        <%s! Icons.magnifying_glass "h-10 w-10" %>
+                    <div class="flex flex-col items-center justify-center pt-24 space-y-4 text-lighter">
+                        <div>
+                            <%s! Icons.magnifying_glass "h-10 w-10" %>
+                        </div>
+                        <div class="text-xl">No Releases found matching "
+                            <%s search %>"</div>
                     </div>
-                    <div class="text-xl">No Releases found matching "<%s search %>"</div>
-                </div>
-                <% | _ -> %>
-                <table class="max-w-5xl lg:max-w-full align-top">
-                    <thead class="bg-body-700 text-white text-left rounded-xl">
-                        <tr>
-                            <th class="py-4 px-6 rounded-l-lg text-x w-3/5">Version</th>
-                            <th class="py-4 px-6">Date</th>
-                            <th class="py-4 px-6 rounded-r-lg">Actions</th>
-                        </tr>
-                    </thead>
-                    <tbody>
-                        <% releases |> List.iter (fun (release : Data.Release.t) -> %>
-                        <tr>
-                            <td class="py-4 px-6 font-semibold">
-                                <div class="font-semibold">
-                                    <%s release.version %>
+                    <% | _ -> %>
+                        <div class="max-w-5xl lg:max-w-full align-top">
+                            <div class="bg-body-700 text-white text-left rounded-xl hidden lg:flex">
+                                <div class="w-2/3">
+                                    <h2 class="py-4 px-6 rounded-l-lg text-x">Version</h2>
                                 </div>
-                                <div class="prose prose-orange font-normal">
-                                    <%s! release.highlights_html %>
+                                <div class="lg:w-1/3 flex">
+                                    <h2 class="py-4 px-6 w-1/2">Date</h2>
+                                    <h2 class="py-4 px-6 rounded-r-lg w-1/2">Actions</h2>
                                 </div>
-                            </td>
-                            <td class="py-4 px-6 font-medium align-top">
-                                <%s release.date %>
-                            </td>
-                            <td class="py-4 px-6 align-top">
-                                <a href="<%s Url.release release.version %>" class="text-primary-600 font-medium block">
+                            </div>
+                            <% releases |> List.iter (fun (release : Data.Release.t) -> %>
+                                <div class="flex flex-col lg:flex-row md:flex-col">
```

Comment:
```suggestion
                                <div class="flex flex-col lg:flex-row">
```

Would collapse on `md` because everything nicely left-aligns and people don't have the primary content floating to the right.